src/cleanupCommand/cleanupManager.ts

```diff
@@ -1,81 +1,178 @@
+      this.logger.info({
+        msg: `Will execute cleanup to ${notRunningExportFilteredBatch.length} success jobs of type: '${swappJobType}'`,
+        batch: `${i + 1}/${Math.floor(notCleanedAndSuccess.length / this.discreteBatchSize + 1)}`,
+      });
+      this.logger.debug({ jobIds: notRunningExportFilteredBatch.map((job) => job.id) });
+      const tilesDirectories = this.getSwappedTilesLocation(notRunningExportFilteredBatch);
+      await this.tileProvider.deleteDiscretes(tilesDirectories);
+
+      // clean source data only for jobs excluded the blacklist
+      const blackListFlitteredBatch =
+        this.sourceBlackList.length > 0 ? this.filterBlackListSourcesTasks(notRunningExportFilteredBatch) : notRunningExportFilteredBatch;
+      const sourcesDirectories = this.getSourcesLocation(blackListFlitteredBatch);
+
+      const sourcesToDelete = notRunningExportFilteredBatch.filter((discrete) => !this.sourceBlackList.includes(discrete.parameters.originDirectory));
+      const ignoredSources = notRunningExportFilteredBatch.filter((discrete) => this.sourceBlackList.includes(discrete.parameters.originDirectory));
+      this.logger.info({
+        msg: `Will execute sources deletion after excluded from blacklist'`,
+        sourcesToDelete: sourcesToDelete.length ? sourcesToDelete.map((source) => source.parameters.originDirectory) : [],
+        ignoredSources: ignoredSources.length ? ignoredSources.map((source) => source.parameters.originDirectory) : [],
+      });
+      await this.sourcesProvider.deleteDiscretes(sourcesDirectories);
+
+      if (notRunningExportFilteredBatch.length) {
+        this.logger.info({
+          msg: `Complete and mark jobs as 'Completed' with file directories remove`,
+          jobIds: notRunningExportFilteredBatch.map((job) => job.id),
+        });
+        await this.jobManager.markAsCompletedAndRemoveFiles(notRunningExportFilteredBatch);
+      }
     }
   }
 
   public async cleanFailedIncomingSyncTasks(): Promise<void> {
     const FAILED_CLEANUP_DELAY = this.config.get<number>('failed_cleanup_delay_days.sync');
     const deleteDate = new Date();
     deleteDate.setDate(deleteDate.getDate() - FAILED_CLEANUP_DELAY);
+    this.logger.info({ msg: `Running Cleanup for failed incoming sync jobs of type: ${this.syncJobType}` });
     const notCleanedAndFailed = await this.jobManager.getFailedAndNotCleanedIncomingSyncJobs();
     for (let i = 0; i < notCleanedAndFailed.length; i += this.discreteBatchSize) {
       const currentBatch = notCleanedAndFailed.slice(i, i + this.discreteBatchSize);
+      this.logger.info({
+        msg: `Will execute cleanup to ${currentBatch.length} failed incoming sync jobs of type: '${this.syncJobType}'`,
+        batch: `${i + 1}/${Math.floor(notCleanedAndFailed.length / this.discreteBatchSize + 1)}`,
+      });
+      this.logger.debug({ jobIds: currentBatch.map((job) => job.id) });
```

Comment:
please enter this debug log to the info log

Reply:
done

Reply:
done